README.md

```diff
@@ -2,136 +2,130 @@
-sed -i "s/02:FF:FF:FF:FF:FF/$mac_oui:FF:FF:FF/" kubemacpool.yaml
-kubectl apply -f ./kubemacpool.yaml
-```
-
-### kubemacpool service
-
-Kubemacpool is set to allocate mac to [supported interfaces](#About) on Pods and VirtualMachines.
-The user can specify whether MAC address will be allocated to all VirtualMachines and Pods by default or not.
-This can be done by patching of the following snippets. Patched manifest will be on [testing manifests](config/test/kubemacpool.yaml):
-
-To set kubemacpool to allocate all VirtualMachines and Pods by default (see [opt-out Mode](#How-to-opt-out-kubemacpool-MAC-assignment-for-a-namespace-in-Opt-out-mode)):
-```bash
-cp config/default/mutatepods_opt_out_patch.yaml config/test/mutatepods_opt_mode_patch.yaml
-cp config/default/mutatevirtualmachines_opt_out_patch.yaml config/test/mutatevirtualmachines_opt_mode_patch.yaml
-make generate-test
-```
-
-To set kubemacpool to not allocate all VirtualMachines and Pods by default (see [opt-in Mode](#How-to-opt-in-kubemacpool-MAC-assignment-for-a-namespace-in-Opt-in-mode))):
-```bash
-cp config/default/mutatepods_opt_in_patch.yaml config/test/mutatepods_opt_mode_patch.yaml
-cp config/default/mutatevirtualmachines_opt_in_patch.yaml config/test/mutatevirtualmachines_opt_mode_patch.yaml
-make generate-test
-```
-
-**note:** The User can of-course set Pods and VirtualMachines to separate opt-modes by sub-setting the above snippets. 
-
-**note:** The default opt-mode for [testing manifests](config/test/kubemacpool.yaml) is opt-out for VirtualMachines and Pods. 
-
-**note:** The default opt-mode for [release manifests](config/release/kubemacpool.yaml) is opt-out for VirtualMachines and opt-out for Pods. 
-
-#### How to opt-out kubemacpool MAC assignment for a namespace in Opt-out mode
-
-You can opt-out MAC assignment on your namespace by adding the following labels:
-- `mutatepods.kubemacpool.io=ignore` - to opt-out MAC assignment for Pods in your namespace
-- `mutatevirtualmachines.kubemacpool.io=ignore` - to opt-out MAC assignment for VMs in your namespace
-
-To disable kubemacpool MAC assignment on a specific namespace:
-```bash
-kubectl label namespace example-namespace mutatepods.kubemacpool.io=ignore mutatevirtualmachines.kubemacpool.io=ignore
-namespace/example-namespace labeled
-```
+For test environment you can choose to deploy this plugin on [local virtualized cluster](docs/deployment-on-local-cluster.md) or on your [arbitrary cluster](docs/deployment-on-arbitrary-cluster.md).
 
-To re-enable kubemacpool MAC assignment in a namespace:
-```bash
-kubectl label namespace example-namespace mutatepods.kubemacpool.io- mutatevirtualmachines.kubemacpool.io-
-namespace/example-namespace labeled
-```
+For Production deployment see this [guide](docs/deployment-on-production-env.md):
 
-#### How to opt-in kubemacpool MAC assignment for a namespace in Opt-in mode
+## Example #1 - Allocating MAC Addresses to a VirtualMachine
```

Comment:
Could you pick a tiny example that would fit one screen and use it in the main README (does not even need to be copy-paste, just to get a feeling for the API) and refer to these full examples stored under docs? The README should not be overwhelming

Reply:
Done.
Added a simple-vm.yaml in the docs folder for quick apply of the example